install_config/install/first_steps.adoc

```diff
@@ -26,13 +26,22 @@ endif::[]
 these sets in the *openshift* project, which is a default project to which all
 users have view access.
 
-If you opted out of creating them during an link:advanced_install.html[advanced
-installation], you can use the following instructions to create the objects
+Use the following instructions to create the objects
 yourself. The files are installed on the file system of your master.
 
+[NOTE]
+====
+This topic is only necessary if you installed OpenShift using a different method
+than the link:quick_install.html[quick installation] or the
```

Comment:
not serious, just a nit for flow:

"using a different method than" --> "using a method other than"